libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -69,6 +74,32 @@ class AC_PrecLand
     // process a LANDING_TARGET mavlink message
     void handle_msg(const mavlink_landing_target_t &packet, uint32_t timestamp_ms);
 
+    // State of the Target Location
+    enum class State: uint8_t {
```

Comment:
I think "State" is perhaps too general and we should call this "LandingTargetState".  If we agree this name is better let's also rename, "get_plnd_target_state()" to "get_landing_target_state()".